module.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: app.fybrik.io/v1beta1
+kind: FybrikModule
+metadata:
+  name: go-module-server
+  labels:
+    name: go-module-server
+    version: latest # semantic version
+spec:
+  type: service
+  chart:
+    name: ghcr.io/aradhalevy/go-module-chart:0.0.0
+    values:
+      image.tag: master
+  capabilities:
+    - capability: read
+      scope: workload
+      api:
+        connection:
+          name: fybrik-go
```

Comment:
I think there is a connection type that can be reused for http requests. 
Also, is this always http or can it be https when running with tls enabled?

Reply:
as we discussed, I'll leave this for now